### PR TITLE
feat(opensuse): add a opensuse tumbleweed reference image

### PR DESCRIPTION
**Pull request by msmeissn**
added a tumbleweed testimage for use in trivy integration testing

### CODE REVIEW

File: copy-images.sh [hunk 1]
```diff
@@ -180,6 +180,17 @@ EOS
...
 
+cat <<EOS
+
+#############################################
+registry.opensuse.org/opensuse/tumbleweed:20240607.30.45
+#############################################
+
+EOS
+
+crane copy registry.opensuse.org/opensuse/tumbleweed@sha256:427d16a14c45614f51357aeebee0dfe209a1cebfc044b3b724b6ea35663b3111 ${TEST_IMAGE}:opensuse-tumbleweed
```
Comment (DmitriyLewen) [inline copy-images.sh:191]:
```bash
➜ crane copy registry.opensuse.org/opensuse/tumbleweed@sha256:427d16a14c45614f51357aeebee0dfe209a1cebfc044b3b724b6ea35663b3111 ghcr.io/dmitriylewen/trivy-test-images:opensuse-tumbleweed
2024/07/04 16:09:26 Copying from registry.opensuse.org/opensuse/tumbleweed@sha256:427d16a14c45614f51357aeebee0dfe209a1cebfc044b3b724b6ea35663b3111 to ghcr.io/dmitriylewen/trivy-test-images:opensuse-tumbleweed
Error: fetching "registry.opensuse.org/opensuse/tumbleweed@sha256:427d16a14c45614f51357aeebee0dfe209a1cebfc044b3b724b6ea35663b3111": GET https://registry.opensuse.org/v2/opensuse/tumbleweed/manifests/sha256:427d16a14c45614f51357aeebee0dfe209a1cebfc044b3b724b6ea35663b3111: MANIFEST_UNKNOWN: 
```
This image doesn't work for me.
Can you test this and create [package](https://github.com/msmeissn?tab=packages&repo_name=trivy-test-images) in your fork to show that it works?

**Review by DmitriyLewen — CHANGES_REQUESTED** [review]

File: copy-images.sh (hunk 1 above)
Comment (msmeissn) [reply copy-images.sh]: ok, i fixed the digest hash now. crane export works when testing

Comment (DmitriyLewen) [reply copy-images.sh]: I can confirm - it works - https://github.com/DmitriyLewen/trivy-test-images/pkgs/container/trivy-test-images/239305417?tag=opensuse-tumbleweed

**Review by DmitriyLewen — APPROVED** [review]
LGTM

@knqyf263 before merge this PR - take a look and merge #33 to fix trigger of copy script

**Pull request merged**